docs/user/01-20-istio-custom-resource.md

```diff
@@ -0,0 +1,336 @@
+The `istios.operator.kyma-project.io` CustomResourceDefinition (CRD) describes the kind and the format of data that Istio Controller uses to configure, update, and manage the Istio installation. Applying the CR triggers the installation of Istio, and deleting it triggers the uninstallation of Istio. To get the up-to-date CRD in the `yaml` format, run the following command:
```

Comment:
The other Markdown files start with a header like `# Istio custom resource`, is this intentionally missing here?

Reply:
It's not, I added it. Thanks!